rancher/v1.6/en/cattle/volumes/index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Volumes in Rancher
+layout: rancher-default-v1.6
+version: v1.6
+lang: en
+---
+
+## Volumes
+---
+
+Persistent volumes are a very important part of a stateful application. Rancher automates the installation and management of Docker volume plugins and supports a few in-house plugins. Each plugin is typically run inside of a container on every host comprising an environment. In Cattle terms, the plugin is a [global service]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/cattle/scheduling/#global-service).
+
+### Terminology
+
+The terms `volume plugin` and `volume driver` are used interchangeably in documentation and reference the same thing. 
+
+The term `storage driver` pertains to how containers and images are stored and managed on your Docker hosts. Examples are: aufs, btrfs, zfs, devicemapper. These drivers are outside the scope of this document and are discussed [here](https://docs.docker.com/engine/userguide/storagedriver/imagesandcontainers/).
+
+### Volume Scope
+
+There are three different scopes for volumes created in Rancher.
```

Comment:
Need to define what volume scope means generally. Something like:
The scope refers to the level at which the volume is managed.